features/facebookImport/test/datasets/post-reactions-data.js

```diff
@@ -5,7 +5,7 @@ import {
 import { createMockedZip } from "../utils/data-creation";
 
 export const DATASET_EXPECTED_VALUES = {
-    numberOfPostsReactions: 6,
+    numberOfPostsReactions: 7,
```

Comment:
Why not compute it from the KNOWN_REACTION_TYPES  constant?

Reply:
For now this is unrelated to `KNOWN_REACTION_TYPES`. There are more reactions having the same type in the dataset.

Reply:
Then what is it related to? My point anyway is that it should be computed, or it will be very difficult to refactor it should the conditions change.

Reply:
For now this is the length of the array returned by `createPostReactionsDataset`. Now the datasets here are manually created.

Reply:
would it be possible then to change it to specifically that?

Reply:
Ideally, for that we need to change more how we handle data the example data. Now the dataset it separated from the related metrics. I changed this but now we create the dataset twice

Reply:
Thanks